apps/ledger-live-mobile/src/components/CustomImage/FramedImageWithLottie.tsx

```diff
@@ -6,14 +6,10 @@ import FramedImage, {
   ImageSourceContext,
 } from "./FramedImage";
 
-/**
- * squareRoot(width of the lottie bounding box / width of the device frame in the lottie)
- * */
-const lottieScale = Math.sqrt(1182 / 502);
-
 const lottiesStyle = {
   width: transferLottieConfig.frameWidth,
-  transform: [{ scale: lottieScale }],
+  height: transferLottieConfig.frameHeight,
+  transform: [{ scale: 2.3 }],
```

Comment:
you forgot the square root no?
the value would be approximately 1.534
<img width="242" alt="Screenshot 2023-05-29 at 18 05 32" src="https://github.com/LedgerHQ/ledger-live/assets/91890529/36157a44-74c9-44e4-aac0-131785a604a4">

Reply:
and could you put back the comment explaining where this value comes from please?

Reply:
I think it broke because of the recent `lottie-react-native` upgrade and the fact that the `height` was missing here.
I remember that in Alex's PR (upgrading react lottie) he had to manually compute the aspect ratio of the lotties.
See this comment: https://github.com/LedgerHQ/ledger-live/pull/3324/files#diff-f90cf65b8af0581f4bf668d6de924b4eddce3ccb352ebd0a0a0b0f3a01cf5e30R31

Reply:
maybe you should double check in the "Custom lock screen assets" debug screen that everything is still matching (that the static assets have the same size and align correctly with the animated assets)

Reply:
I have no idea, but 1.534 is way too small and didnt work. 
Feel free to take a look if you can figure it out 🙏🏼